### PR TITLE
[kube-prometheus-stack] Add hint for zero downtime migration from stable/prometheus-operator

### PR DESCRIPTION
**Pull request by raffis**
Signed-off-by: Raffael Sahli <raffael.sahli@doodle.com>

#### What this PR does / why we need it:
The current description to migrate from stable/prometheus-operator only covers a migration path with downtime. In most cases people want a zero downtime migration.

This fairly small pr adds a simple hint to cover the fullnameOverride option.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by Xtigyro** [comment]
Hey @raffis 👋 Thank you very much for the contribution - it is appreciated!

Could you please fix the linting errors?

```
 2020-10-21 11:42:29 [ERROR ]   Found errors in [markdownlint] linter!
2020-10-21 11:42:29 [ERROR ]   [charts/kube-prometheus-stack/README.md:230 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]]
2020-10-21 11:42:29 [ERROR ]   Linter CMD:[markdownlint -c /action/lib/.automation/.markdown-lint.yml charts/kube-prometheus-stack/README.md]
2020-10-21 11:42:32 [WARN  ]   No .editorconfig found at:[/github/workspace/.editorconfig]
```

File: charts/kube-prometheus-stack/README.md [hunk 1]
```diff
@@ -220,6 +220,19 @@ To do so, you can set `prometheus.prometheusSpec.podMonitorSelectorNilUsesHelmVa
...
 ## Migrating from stable/prometheus-operator chart
 
+## Zero downtime
+
+Since `kube-prometheus-stack` is fully compatible with the `stable/prometheus-operator` chart a migration without downtime can be achieved.
+However the old name prefix needs to be kept. If you want the new name please follow the step by step guide bellow (with downtime).
+
+You can override the fullname to achieve this:
+
+```
```
Comment (Xtigyro) [inline charts/kube-prometheus-stack/README.md:230]: you need to specify your language on this line.

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/README.md (hunk 1 above)
Comment (raffis) [reply charts/kube-prometheus-stack/README.md:232]: Done

**Review by bismarck — APPROVED** [review]
lgtm

**Pull request merged**